test/resources/billing.test.js

```diff
@@ -2,38 +2,73 @@
 import { expect } from 'chai';
 
 import EasyPost from '../../src/easypost';
-import * as setupPolly from '../helpers/setup_polly';
+import {
+  MockMiddleware,
+  MockRequest,
+  MockRequestMatchRule,
+  MockRequestResponseInfo,
+} from '../helpers/mocking';
+
+const middleware = (request) => {
+  return new MockMiddleware(request, [
+    new MockRequest(
+      new MockRequestMatchRule('POST', 'v2\\/bank_accounts\\/\\S*\\/charges$'),
+      new MockRequestResponseInfo(200, {}),
+    ),
+    new MockRequest(
+      new MockRequestMatchRule('POST', 'v2\\/credit_cards\\/\\S*\\/charges$'),
+      new MockRequestResponseInfo(200, {}),
+    ),
+    new MockRequest(
+      new MockRequestMatchRule('DELETE', 'v2\\/bank_accounts\\/\\S*$'),
+      new MockRequestResponseInfo(200, {}),
+    ),
+    new MockRequest(
+      new MockRequestMatchRule('DELETE', 'v2\\/credit_cards\\/\\S*$'),
+      new MockRequestResponseInfo(200, {}),
+    ),
+    new MockRequest(
+      new MockRequestMatchRule('GET', 'v2\\/payment_methods$'),
+      new MockRequestResponseInfo(200, {
+        id: 'summary_123',
+        primary_payment_method: {
+          id: 'card_123',
+          last4: '1234',
+        },
+        secondary_payment_method: {
+          id: 'bank_123',
+          bank_name: 'Mock Bank',
+        },
+      }),
+    ),
+  ]);
+};
 
 describe('Billing Resource', function () {
-  setupPolly.startPolly();
 
   before(function () {
-    this.easypost = new EasyPost(process.env.EASYPOST_PROD_API_KEY);
+    this.easypost = new EasyPost(process.env.EASYPOST_TEST_API_KEY, {
+      requestMiddleware: middleware,
+    });
   });
 
   beforeEach(function () {
-    const { server } = this.polly;
-    setupPolly.setupCassette(server);
   });
 
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip('fund a wallet', async function () {
-    // Skipping due to the lack of an available real payment method in tests
+  it('fund a wallet', async function () {
     const fund = await this.easypost.Billing.fundWallet('2000', 'primary');
 
     expect(fund).to.equal(true);
   });
 
-  // eslint-disable-next-line jest/no-disabled-tests
-  it.skip('delete a payment method', async function () {
+  it('delete a payment method', async function () {
     // Skipping due to the lack of an available real payment method in tests
```

Comment:
There are still comments here to skip these. Please check the whole file

Reply:
Resolved